migrations/v4.js

```diff
@@ -0,0 +1,31 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('Matching - v3.0.0 to v4.2.0', async () => {
+  let matchings, course, courseMatchingGlobals;
+  whereFromPlugin('Matching - from v3.0.0', { name: 'adapt-contrib-matching', version: '<4.2.0' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add globals ariaCorrectAnswer attribute', async content => {
+    course = content.find(({ _type }) => _type === 'course');
```

Comment:
Consider adding a check for existence of globals (see https://github.com/adaptlearning/adapt-contrib-narrative/blob/issue/318/migrations/v4.js#L20)